### PR TITLE
(Dash) Rearrange elements on the page to fit #250 issue

### PR DESCRIPTION
**Pull request by Zedmor**
Adds the header element for a sidebar.

Screenshot from 2020-03-27 21-59-43

### CODE REVIEW

**Comment by quinn-dougherty** [comment]
Closes https://github.com/CodeForPhilly/chime/issues/250

**Comment by quinn-dougherty** [comment]
i checked it out locally -- functionality is there! wanna wait for Brian or Christopher to review before merging, tho.

**Review by BrianThomasRoss — APPROVED** [review]
Looks good

**Pull request merged**
